airflow/www/templates/airflow/tree.html

```diff
@@ -87,376 +92,8 @@
   {{ super() }}
   <script src="{{ url_for_asset('d3.min.js') }}"></script>
   <script src="{{ url_for_asset('d3-tip.js') }}"></script>
-  <script src="{{ url_for_asset('taskInstances.js') }}"></script>
```

Comment:
Was this script removed on purpose?

Reply:
Yes. It's being imported into `tree.js` instead.

Reply:
Ah, I was skimming for `taskInstances`  not `task_instances`.